core/src/test/scala/org/typelevel/paiges/PaigesTest.scala

```diff
@@ -254,4 +254,36 @@ the spaces""")
   test("cat") {
     assert(Doc.cat(List("1", "2", "3") map Doc.text).render(80) == "123")
   }
+
+  test("Scala whitespace-sensitivity 1") {
+    import Document.ops._
+    import Doc._
+
+    val origStr =
+      """y match {
+        |  case foo => 5
+        |  case bar => 6
+        |}""".stripMargin
+
+    val case1 = "case".doc :& "foo" :& "=>" & 5.doc
+    val case2 = "case".doc :& "bar" :& "=>" & 6.doc
+    val inner = (case1 + hardLine + case2).aligned
+    val orig = "y".doc :& "match" & "{".doc + (lineOrSpace + inner).nested(2) + lineOrSpace + "}".doc
+    val origStr1 = orig.render(6)
+    assert(origStr1 == origStr)
+
+    val origStrWide =
+      """y match { case foo => 5
+        |          case bar => 6 }""".stripMargin
+
+    val origStrWide1 = orig.render(100)
+    assert(origStrWide1 == origStrWide)
+
+    val flatStr =
```

Comment:
isn't this the same as `origStrWide`?

Reply:
I cleaned this up.  Thanks for pointing out the sloppiness.